IdentityCore/src/cache/MSIDMacTokenCache.m

```diff
@@ -276,6 +276,14 @@ - (MSIDAppMetadataCacheItem *)appMetadataWithKey:(MSIDCacheKey *)key
     return nil;
 }
 
+- (NSArray<MSIDAppMetadataCacheItem *> *)appMetadataEntriesWithKey:(MSIDCacheKey *)key
+                                                        serializer:(id<MSIDAppMetadataItemSerializer>)serializer
+                                                           context:(id<MSIDRequestContext>)context
+                                                             error:(NSError **)error
+{
+    return nil;
```

Comment:
can you put a little comment on why perhaps?
or "todo" if otherwise, just in case.

Reply:
Sure. Will do.